tcms/bugs/tests/test_views.py

```diff
@@ -0,0 +1,61 @@
+# pylint: disable=too-many-ancestors
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from tcms.tests.factories import BugFactory
+from tcms.tests import BaseCaseRun, user_should_have_perm
+from tcms.utils.permissions import initiate_user_with_default_setups
+
+
+class TestBugStatusChange(BaseCaseRun):
+    """Test the possible bug status changes.
+
+    Cases:
+    * Closing an open bug.
+    * Reopening a closed bug.
+    """
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        initiate_user_with_default_setups(cls.tester)
+        cls.comment_bug_url = reverse('bugs-comment')
+        user_should_have_perm(cls.tester, 'bugs.change_bug')
+
+    def test_close_an_open_bug(self):
+        bug = BugFactory(status=True)
+
+        edit_bug_data = {
+            'bug': bug.pk,
+            'text': 'Close the bug.',
+            'action': 'close'
+        }
+
+        redirect_url = reverse('bugs-get', args=[bug.pk])
+        response = self.client.post(self.comment_bug_url, edit_bug_data, follow=True)
+
+        self.assertContains(response, _('*bug closed*').replace('*', ''))
```

Comment:
Question:

_('bug closed') could not be translated but  _('*bug closed*') is translated. And the response returns <em>bug ferme<em> (french), so as a workaround i have deleted asteriks here.

I am not sure if this is the intended behaviour.

Reply:
Leave back the asteriks characters. Translation is done verbatim and the source string is `*bug closed*`. If you had a test failing here then the reason is not that it can't be translated but precisely because it can.

The translation is `*bug fermé*` where the last character is unicode and is returned as an unicode sequence inside the HTML. For such characters we've got a slightly different way to mach the expected. See html.unescape in testruns/tests/test_views.py

Reply:
No, the problem is asteriks,they transform into `<em></em>` in response html, so there is no asteriks but `<em>bug fermé</em>`. There is no problem with `é` .

Reply:
See `markdown2html` in `tcms/core/templatetags/extra_filters.py` then

Reply:
Yep thanks for the hint, used it to parse the string for assert.